Trannet/Models/Models.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Trannet.Services;
+public record StopTimeResponse(string stop_id, string arrival_time, string departure_time);
```

Comment:
These should be record struct, as they were previously structs, otherwise, it's going to allocate a hell of managed objects... pretty unfortunate considering that your PR is meant to reduce managed allocations! 😉